code/modules/client/preferences.dm

```diff
@@ -35,7 +36,8 @@ var/global/list/special_role_times = list( //minimum age (in days) for accounts
 	num2text(BE_ALIEN) = 21,
 	num2text(BE_NINJA) = 21,
 	num2text(BE_MUTINEER) = 21,
-	num2text(BE_MALF) = 30
+	num2text(BE_MALF) = 30,
+	num2text(BE_REVENANT) = 30

```

Comment:
Really dont' think that someone should have to wait 30 days to play revenant; a mere 7-10 days would be plenty.